README.md

```diff
@@ -264,12 +275,14 @@ mv docopts-32bit docopts
 cp docopts docopts.sh /usr/local/bin
 ```
 
-You are strongly encouraged to build your own binary. Find a local golang developper in whom you trust and ask her, in exchange for a beer or two, if she could build it for you. ;)
-
+You are strongly encouraged to build your own binary, which is easy once
+you have Go installed. Or find a local golang developer that you
+trust and ask her, in exchange for a beer or two, if she could build it for you. ;)
 
 ## Compiling
 
-Requires a directory to use as [Go workspace](https://golang.org/doc/code.html#Organization).
+Requires a directory to use as a [Go workspace](https://golang.org/doc/code.html#Organization),
+such as `~/gocode`.
```

Comment:
is `~/gocode` a common place? Mine is `~/code/go` I'm not sure we should re-document installing a Go workspace here. As it may become deprecated in time.

Reply:
Removing this change.